ThreadX_RTApp_MT3620_AzureRTOS/README.md

```diff
@@ -0,0 +1,35 @@
+## Overview
+
+This sample demonstrates how Azure RTOS and Codethink driver library are able to run together on MT3620.
+
+This sample app for an MT3620 real-time core running Azure RTOS that creates several threads and repeatedly transmits statistic message over a UART. These messages can be read in terminal application on a computer using a USB-to-serial adapter. By default, it uses the real-time core's dedicated UART, but if your hardware doesn't expose this UART's TX pin, then the sample can be altered to use a different UART.
```

Comment:
Can we have this split on a max line length of ~100.

Reply:
Just want to confirm you want each line in source file is less than 100 columns right? Not the final markdown file display on browser (a single line break has no effect in markdown)

Reply:
Yes. It's easier for editors that don't handle markdown as gracefully as others out of the box (visual studio for example).